crates/http-sink/src/sink.rs

```diff
@@ -36,15 +36,16 @@ impl Sink<String> for HttpSink {
         let unfold = futures::sink::unfold(
             request,
             |mut request: RequestBuilder, record: String| async move {
-                trace!("{:?}", request);
+                tracing::trace!("{:?}", request);
 
                 request = request.body(record);
                 let response = request
                     .try_clone()
                     .ok_or(anyhow!("ERR: Cannot clone request"))?
                     .send()
                     .await?;
-                trace!("{:?}", response);
+                tracing::trace!("{:?}", response);
+                tracing::info!("Response Status: {}", response.status());
```

Comment:
@sehz suggested to put status only if it is not 200, I guess that in that case, then we would need it if response is not in the 200-299 range

Reply:
maybe with a warn if not in that range?